### PR TITLE
feat: support limit without orderby

### PR DESCRIPTION
**Pull request by dk-lockdown**
ref: https://github.com/cectc/dbpack/issues/168

### Ⅰ. Describe what this PR did
is sharding mode, support query with limit, like:
```
select id, drug_res_type_id, base_type, sale_price from drug_resource where id between ? and ? limit ?,?
```

### Ⅳ. Describe how to verify it
check ci result

### CODE REVIEW

File: cmd/cmd.go [hunk 1]
```diff
@@ -155,13 +154,12 @@ var (
...
 			}
 
-			tracingMgr, err := tracing.NewTracer(Version, "console")
-			if err != nil {
-				log.Fatalf("could not setup tracing manager: %s", err.Error())
-			}
-			if err != nil {
-				log.Fatalf("could not setup tracing exporter: %s", err.Error())
-			}
+			// temporarily turn off tracer output
```
Comment (bohehe) [inline cmd/cmd.go:157]: Why need to temporarily turn off this trace output?

**Review by bohehe — APPROVED** [review]
LGTM

File: cmd/cmd.go (hunk 1 above)
Comment (dk-lockdown) [reply cmd/cmd.go:157]: Too much trace information is printed to the console, making it inconvenient to view the log.The initialization of tracer will also change in the future.

Comment (bohehe) [reply cmd/cmd.go:157]: Thanks, understood

**Pull request merged**